### DISCUSSION TITLE
Question about monitoring multiple harvester

### DISCUSSION THREAD

**Discussion by fireboy0526** [martomi/chiadog#129] [Q&A] [answered]
Hello there,

Just wanted to say that I love this project and have been using it from day 1, thanks dev.

I would like to ask about monitoring multiple harvester with chiadog. The OS I'm using is ubuntu 20.04.02 server.

In the wiki, it states that:
`Copy config.yaml into multiple configs for each remote harvester`

and this line too:
`Start chiadog for each harvester in a separate terminal.`

Does this means that I'll have to duplicate multiple chiadog program, each with a different config file for each harvester, and then start up each individual chiadog program?

If not, how can we get chiadog to startup with ./start.sh (config file to load?) with only just one chiadog program?

Thanks!

**Comment by pieterhelsen** [comment, marked as answer]
I generally use tmux to handle my separate chiadog sessions.

- `tmux new -s chiadog`
- CTRL B + , (release ctrl for the comma) > rename a window (ex. Harvester1)
- `python3 main.py --config harvester1.yaml` (make sure to setup the venv)
- CTRL B + C > create new window
- rename to Harvester2
- `python3 main.py --config harvester2.yaml`

And so on... you could write your own start.sh that does all of this for you

**Reply by fireboy0526** [reply]
> I generally use tmux to handle my separate chiadog sessions.
> 
>     * `tmux new -s chiadog`
> 
>     * CTRL B + , (release ctrl for the comma) > rename a window (ex. Harvester1)
> 
>     * `python3 main.py --config harvester1.yaml` (make sure to setup the venv)
> 
>     * CTRL B + C > create new window
> 
>     * rename to Harvester2
> 
>     * `python3 main.py --config harvester2.yaml`
> 
> 
> And so on... you could write your own start.sh that does all of this for you

Thank you for the answer! This should clear up everything for me.

By the way,  what's the advantage of using tmux over screen?

**Comment by pieterhelsen** [comment]
Flavour really 😄 
Some people swear by tmux, but use whichever fits your needs. I quite like the shortcuts and interface.

**Reply by fireboy0526** [reply]
I see. I will definitely give tmux a try!

Thanks very much for your help!
